src/wolframite/base/package.clj

```
@@ -0,0 +1,66 @@
+(ns wolframite.base.package
+  "The place to be for loading and manipulating Wolfram packages.
+"
+  (:require
+   [clojure.string :as str]
+   [babashka.fs :as fs]
+   [wolframite.core :as wl]
+   [wolframite.impl.wolfram-syms.intern :as wi]
+   [wolframite.wolfram :as w]))
+
+(defn intern-context!
+  "Finds all of the public symbols in the given context and interns them to a clojure-accesible namespace. This is put under the given symbol or the context by default.
+
+  N.B. context does not contain the trailing '`'"
+  ([context]
+   (intern-context! context (symbol context)))
+  ([context alias]
```

```suggestion
  ([context alias]
    {:pre [(string? context) (symbol? alias)]}
```
let's be defensive and fail early